docs/src/dictvectors.md

```diff
@@ -19,6 +19,9 @@ deposit!
 storage
 freeze
 localpart
+fciqmc_step!
```

Comment:
If `fciqmc_step` lives in `Interfaces` now then perhaps there should be a reference to it on the doc page, which I think is just the docstring for the `Interfaces` module